File: 3606-minimum-element-after-replacement-with-digit-sum/minimum-element-after-replacement-with-digit-sum.py
class Solution:
    def minElement(self, nums: List[int]) -> int:
        n = len(nums)
        ans = []

        for num in nums:
            x = str(num)
            tsum = 0
            for i in x:
                tsum += int(i)

            ans.append(tsum)

        return min(ans)

File: 3606-minimum-element-after-replacement-with-digit-sum/minimum-element-after-replacement-with-digit-sum.cs
public class Solution {
    public int MinElement(int[] nums) {
        int ans = int.MaxValue;

        foreach (int num in nums) {
            int x = num, sum = 0;

            while (x > 0) {
                sum += x % 10;
                x /= 10;
            }

            ans = Math.Min(ans, sum);
        }

        return ans;
    }
}